### PR TITLE
Web Device API reference update for Tizen 3.0

### PR DESCRIPTION
**Pull request by pwasowski2**
### Change Description ###

This commit updates and unifies style and formatting of Web Device API reference descriptions and code examples for Tizen 3.0.

### CODE REVIEW

**Comment by pwasowski2** [comment]
Hyperlinks to _Managing Bluetooth Tutorial_ and _Managing NFC Tutorial_ have been updated.

File: docs/application/web/api/3.0/device_api/tv/index.html [hunk 1]
```diff
@@ -1,358 +1,274 @@
...
-            <h4 id="Tizen">Base</h4>
-             <table class="api-list">
-                <thead>
-                <tr><th>API</th><th>DESCRIPTION</th><th>VERSION</th><th>STATE</th></tr>
-                </thead>
-                <tbody>
-                <tr>
-                <td>
-                <a href="tizen/archive.html" data-origin="tizen" data-domain="filesystem" data-category="core" data-status="D" data-version="">
-                        Archive</a></td>
```
Comment (mijin85cho) [inline docs/application/web/api/3.0/device_api/tv/index.html]: Question : It looks like "Archive" is removed from the index.html file. Would you please confirm?

Comment (pwasowski2) [reply docs/application/web/api/3.0/device_api/tv/index.html]: It was an error. I have added _archive_ to the proper table in the patch set I have just pushed.
Thank you for spotting the problem.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/415

**Comment by mijin85cho** [comment]
Because of the schedule, I merge this PR without confirmation from API team.
If you find an issue in the future, please raise PRs.

**Pull request merged**